pytorch_pfn_extras/onnx/pfto_exporter/export.py

```diff
@@ -124,17 +124,17 @@ def onnx_node_doc_string(onnx_node: torch._C.Node, torch_node: torch._C.Node) ->
 
 
 torch_dtype_to_onnx_data_type = {
-    torch.float32: onnx.TensorProto.DataType.FLOAT,
-    torch.uint8: onnx.TensorProto.DataType.UINT8,
-    torch.int8: onnx.TensorProto.DataType.INT8,
-    torch.int16: onnx.TensorProto.DataType.INT16,
-    torch.int32: onnx.TensorProto.DataType.INT32,
-    torch.int64: onnx.TensorProto.DataType.INT64,
-    torch.bool: onnx.TensorProto.DataType.BOOL,
-    torch.float64: onnx.TensorProto.DataType.DOUBLE,
-    torch.float16: onnx.TensorProto.DataType.FLOAT16,
-    torch.complex64: onnx.TensorProto.DataType.COMPLEX64,
-    torch.complex128: onnx.TensorProto.DataType.COMPLEX128,
+    torch.float32: onnx.TensorProto.DataType.FLOAT,  # type: ignore[attr-defined]
```

Comment:
I wish this was supported in onnx... https://github.com/protocolbuffers/protobuf/issues/8175